script/server

```diff
@@ -7,6 +7,11 @@ set -e
 
 cd "$(dirname "$0")/.."
 
+if [ -f "$(which docker-compose)" ]; then
+  echo "===> Stopping existing containers..."
+  docker-compose stop
+fi
```

Comment:
Maybe here, if docker-compose isn't installed, we can print a helpful message and `exit 1`?

Reply:
Technically, this script is run after `script/setup` which by running `script/bootstrap` checks if docker-compose is installed and returns a message if it's not installed. 
https://github.com/education/classroom/blob/f0ca7f98d62c8c2a92f12a748b03a6f9ad4b770c/script/bootstrap#L53

It can happen if someone doesn't run script in the right order though 🤔 I can add it if you think it'll be helpful.

Reply:
Ooh ok, no this is probably ok then.